### PR TITLE
feat: add new Piscary Enchantment to max enchant list

### PR DESCRIPTION
**Pull request by DuckySoLucky**
## Description
Adds new Enchantment which has been added in [Moby's Fishing Speed Chumpdate](https://hypixel.net/threads/september-6-mobys-fishing-speed-chumpdate.5103151/)

<details>
   <summary>Piscary Enchantment NBT</summary>
   
   ````js
{
    id: "minecraft:enchanted_book",
    Count: 1b,
    tag: {
        display: {
            Lore: ["§7Grants §b+6☂ Fishing Speed§7."],
            Name: "§9Piscary VI"
        }
    },
    Damage: 0s
}
   ````

</details>

### CODE REVIEW

**Review by metalcupcake5 — APPROVED** [review]
didnt test

File: common/constants/enchantments.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ export const MAX_ENCHANTS = new Set([
   "Overload V",
   "Piercing I",
   "Power VII",
+  "Piscary VI",
```
Comment (metalcupcake5) [inline common/constants/enchantments.js:60]: move it up one to make it alphabetical

**Review by metalcupcake5 — CHANGES_REQUESTED** [review]

**Review by metalcupcake5 — APPROVED** [review]
lgtm didnt test

**Review by dukio — APPROVED** [review]
lgtm didn't test

**Review by MartinNemi03 — APPROVED** [review]
lgtm, didn't test

**Pull request merged**